src/commands/api-mesh/describe.js

```diff
@@ -52,7 +52,13 @@ class DescribeCommand extends Command {
 					this.log('Workspace ID: %s', workspaceId);
 					this.log('Mesh ID: %s', meshId);
 
-					const { meshURL } = await getMesh(imsOrgId, projectId, workspaceId, meshId);
+					const { meshURL } = await getMesh(
+						imsOrgId,
+						projectId,
+						workspaceId,
+						workspaceName,
+						meshId,
+					);
```

Comment:
Since the TIDelegator is always returning the url you do not need this check anymore

Reply:
Removed the not needed checks. Now in case of nonTI users TI deligator will not return anything and CLI will check and append the api_key for nonTI users.